### ISSUE TITLE
[VFP]Attributes cannot be defined in CLASS Statement (FoxPro dialect). 

### ISSUE THREAD

**Issue by vfp9** [X-Sharp/XSharpPublic#1566]
A VFP class library project has been created in VS 2022 based on the template with the following code:
```foxpro
Using System
Using System.Collections.Generic
Using System.Linq
Using System.Text

Using System.Drawing
Using System.Windows.Forms

Begin Namespace my
   [ToolboxBitmap(Typeof(myCommandButton), "Button.bmp")];
    Define Class myCommandButton As Button
    Enddefine
End Namespace
```
Error XS9002 Parser: unexpected input 'Define' when build project.

**Comment by RobertvanderHulst** [comment]
Confirmed. "DEFINE" is a so called positional keyword. It is recognized at the start of a line, after Modifiers (such as STATIC or PUBLIC) and after the END keyword. It should also be recognized after the closing bracket `]`.
I'll include the fix for this in 2.21.
For now, I would recommend using the core class syntax (CLASS .. ENDCLASS, and CONSTRUCTOR instead of INIT)

Robert

**Comment by cpyrgas** [comment]
Confirmed fixed
